i18n/states.php

```diff
@@ -292,6 +292,56 @@
 	'CZ' => array(),
 	'DE' => array(),
 	'DK' => array(),
+	'DZ' => array(
+		'DZ-01' => __( 'Adrar', 'woocommerce' ),
+		'DZ-02' => __( 'Chlef', 'woocommerce' ),
+		'DZ-03' => __( 'Laghouat', 'woocommerce' ),
+		'DZ-04' => __( 'Oum El Bouaghi', 'woocommerce' ),
+		'DZ-05' => __( 'Batna', 'woocommerce' ),
+		'DZ-06' => __( 'Béjaïa', 'woocommerce' ),
+		'DZ-07' => __( 'Biskra', 'woocommerce' ),
+		'DZ-08' => __( 'Bechar', 'woocommerce' ),
+		'DZ-09' => __( 'Blida', 'woocommerce' ),
+		'DZ-10' => __( 'Bouira', 'woocommerce' ),
+		'DZ-11' => __( 'Tamanrasset', 'woocommerce' ),
+		'DZ-12' => __( 'Tebessa', 'woocommerce' ),
+		'DZ-13' => __( 'Tlemcen', 'woocommerce' ),
+		'DZ-14' => __( 'Tiaret', 'woocommerce' ),
+		'DZ-15' => __( 'Tizi Ouzou', 'woocommerce' ),
+		'DZ-16' => __( 'Alger', 'woocommerce' ),
+		'DZ-17' => __( 'Djelfa', 'woocommerce' ),
+		'DZ-18' => __( 'Jijel', 'woocommerce' ),
+		'DZ-19' => __( 'Setif', 'woocommerce' ),
+		'DZ-20' => __( 'Saida', 'woocommerce' ),
+		'DZ-21' => __( 'Skikda', 'woocommerce' ),
+		'DZ-22' => __( 'Sidi bel abbes', 'woocommerce' ),
+		'DZ-23' => __( 'Annaba', 'woocommerce' ),
+		'DZ-24' => __( 'Guelma', 'woocommerce' ),
+		'DZ-25' => __( 'Constantine', 'woocommerce' ),
+		'DZ-26' => __( 'Medea', 'woocommerce' ),
+		'DZ-27' => __( 'Mostaghanem', 'woocommerce' ),
+		'DZ-28' => __( 'Msila', 'woocommerce' ),
+		'DZ-29' => __( 'Mascara', 'woocommerce' ),
+		'DZ-30' => __( 'Ouargla', 'woocommerce' ),
+		'DZ-31' => __( 'Oran', 'woocommerce' ),
+		'DZ-32' => __( 'El Bayadh', 'woocommerce' ),
+		'DZ-33' => __( 'Illizi', 'woocommerce' ),
+		'DZ-34' => __( 'Bordj Bou Arreridj', 'woocommerce' ),
+		'DZ-35' => __( 'Boumerdes', 'woocommerce' ),
+		'DZ-36' => __( 'El Taref', 'woocommerce' ),
+		'DZ-37' => __( 'Tindouf', 'woocommerce' ),
+		'DZ-38' => __( 'Tissemsilt', 'woocommerce' ),
+		'DZ-39' => __( 'El Oued', 'woocommerce' ),
+		'DZ-40' => __( 'Khenchla', 'woocommerce' ),
+		'DZ-41' => __( 'Souk Ahras', 'woocommerce' ),
+		'DZ-42' => __( 'Tipaza', 'woocommerce' ),
+		'DZ-43' => __( 'Mila', 'woocommerce' ),
+		'DZ-44' => __( 'Ain Defla', 'woocommerce' ),
+		'DZ-45' => __( 'Naama', 'woocommerce' ),
```

Comment:
```suggestion
		'DZ-45' => __( 'Naâma', 'woocommerce' ),
```

Reply:
@Fcmam5 thanks for your help, but I'll keep `Naama` what is the accepted value for the CLDR version 37 list, see: https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1176-L1224